### DISCUSSION TITLE
Rigid bodies breaking apart during simulation

### DISCUSSION THREAD

**Discussion by annaashki** [glotzerlab/hoomd-blue#1705] [Support] [answered]
basic PBC query - resolved

**Comment by erjank** [comment, marked as answer]
Hello annashki,
The simulation volumes in HOOMD (and other engines) avoid problems with
surfaces--and enable the representation of a bulk material with a finite
number of simulation elements--using *periodic boundary conditions*:
Particles are supposed to come in from the left after exiting to the right.
Allen and Tildesley or Frenkel and Smit are two great texts to dig into for
more details.
Best,
Eric

**Reply by annaashki** [reply]
hi! ahh okay thank you, my first thought was periodic b.c.'s as well but the particles nominally belonging to the same rigid body threw me off. accounting for this should resolve the issue I was having with my calculations, I will try in the morning and update!
UPD: yes that was it, thanks again!
